### DISCUSSION TITLE
Web version of the app

### DISCUSSION THREAD

**Discussion by drjgouveia** [nylo-core/nylo#122] [Q&A] [answered]
Hi! I have a doubt. I can't load the Nylo app in a platform like a web browser or a Linux desktop app. Is it a me problem or is it a thing with Nylo? Is there any workaround?

**Comment by agordn52** [comment]
Hi @drjgouveia,

What problem are you encountering then building Nylo for web?

**Comment by jet10000** [comment]
<img width="594" alt="image" src="https://github.com/nylo-core/nylo/assets/2258120/2f614fa7-ccf7-40e6-b42e-dc0412152c8a">

**Reply by agordn52** [reply, marked as answer]
Hi @jet10000 & @drjgouveia,

Thanks for reporting this, it's resolved now in the latest version [5.25.5](https://pub.dev/packages/nylo_framework)
